lang/ca.json

```diff
@@ -720,8 +720,8 @@
   "Expense.Attachments": "Adjunts",
   "Expense.By": "by {name}",
   "expense.category": "categoria",
-  "expense.created": "Your expense has been submitted with success. It is now pending approval from one of the core contributors of the collective. You will be notified by email once it has been approved. Then, the host ({host}) will proceed to reimburse your expense.",
-  "expense.created.noHost": "Your expense has been submitted with success. It is now pending approval from one of the core contributors of the collective. You will be notified by email once it has been approved.",
+  "expense.created": "Your expense has been submitted with success. It is now pending approval from one of the team members of the Collective. You will be notified by email once it has been approved. Then, the host ({host}) will proceed to reimburse your expense.",
```

Comment:
I think this should say "admins" instead, because that's the role that approves expenses actually.